pipeline/http_post.go

```diff
@@ -14,48 +21,61 @@ package pipeline
 type HTTPPostNode struct {
 	chainnode
 
-	HTTPPostHandlers []*HTTPPostHandler `tick:"Endpoint"`
+	// tick:ignore
+	HTTPPostEndpoints []*HTTPPostEndpoint `tick:"Endpoint"`
 
 	// tick:ignore
-	Url string
+	urls []string
 }
 
 func newHTTPPostNode(wants EdgeType, urls ...string) *HTTPPostNode {
-	var url string
-	// TODO: What to do if len(urls) > 1?
-	if len(urls) != 0 {
-		url = urls[0]
-	}
 	return &HTTPPostNode{
 		chainnode: newBasicChainNode("http_post", wants, wants),
-		Url:       url,
+		urls:      urls,
 	}
 }
 
+// tick:ignore
+func (p *HTTPPostNode) Validate() error {
```

Comment:
This should be named `validate` in order to be automatically called in the parsing execution steps.